docs/.vuepress/plugins/nodes.js

```diff
@@ -40,7 +40,14 @@ async function getNodes() {
 
 	const descriptions = {};
 	for (let name in nodes) {
-		const node = nodes[name].description;
+		let node;
+
+		if (nodes[name].nodeVersions) {
+			const maxVersion = Math.max(...Object.keys(nodes[name].nodeVersions));
```

Comment:
for documentation reasons, should not we be supporting versions? if operations changed between versions, how do we point users to previous version?

Reply:
We should definitely document versions properly at some point. This is a hotfix to unlock the build.